frontend/lib/laletterbuilder/letter-builder/welcome.tsx

```diff
@@ -1,13 +1,27 @@
+import { Trans } from "@lingui/macro";
 import React from "react";
 import { Link } from "react-router-dom";
+import { SimpleClearAnonymousSessionButton } from "../../forms/clear-anonymous-session-button";
 import { ProgressStepProps } from "../../progress/progress-step-route";
+import Page from "../../ui/page";
+import { LALetterBuilderRoutes } from "../route-info";
 import { assertNotNull } from "@justfixnyc/util";
 
 export const LALetterBuilderWelcome: React.FC<ProgressStepProps> = (props) => {
   return (
-    <p>
-      TODO: Add welcome content here.{" "}
-      <Link to={assertNotNull(props.nextStep)}>Next</Link>
-    </p>
+    <Page title="Build your letter" className="content" withHeading="big">
```

Comment:
btw I ALWAYS forget to internationalize these page title strings! obvi don't need to do now but just wanted to flag